arrangements:
  nothing:
    objects: []

  brown_cardboard_box3_nominal:
    objects:
      -
        name: box
        type: brown_cardboard_box3
        parent: ee
        offset:
          x: 0.065
    contacts:
      -
        first: ee
        second: box
        mu: 0.2
        support_area_inset: 0.0

  brown_cardboard_box3_robust:
    objects:
      -
        name: box1
        type: brown_cardboard_box3_1
        parent: ee
        offset:
          x: 0.065
      -
        name: box2
        type: brown_cardboard_box3_2
        parent: ee
        offset:
          x: 0.065
      -
        name: box3
        type: brown_cardboard_box3_3
        parent: ee
        offset:
          x: 0.065
      -
        name: box4
        type: brown_cardboard_box3_4
        parent: ee
        offset:
          x: 0.065
    contacts:
      -
        first: ee
        second: box1
        mu: 0.3
        support_area_inset: 0.005
      -
        first: ee
        second: box2
        mu: 0.3
        support_area_inset: 0.005
      -
        first: ee
        second: box3
        mu: 0.3
        support_area_inset: 0.005
      -
        first: ee
        second: box4
        mu: 0.3
        support_area_inset: 0.005

  brown_cardboard_box2_nominal:
    objects:
      -
        name: box
        type: brown_cardboard_box2
        parent: ee
        offset:
          x: 0.065
    contacts:
      -
        first: ee
        second: box
        mu: 0.2
        support_area_inset: 0.0

  brown_cardboard_box2_robust:
    objects:
      -
        name: box1
        type: brown_cardboard_box2_1
        parent: ee
        offset:
          x: 0.065
      -
        name: box2
        type: brown_cardboard_box2_2
        parent: ee
        offset:
          x: 0.065
      -
        name: box3
        type: brown_cardboard_box2_3
        parent: ee
        offset:
          x: 0.065
      -
        name: box4
        type: brown_cardboard_box2_4
        parent: ee
        offset:
          x: 0.065
    contacts:
      -
        first: ee
        second: box1
        mu: 0.3
        support_area_inset: 0
      -
        first: ee
        second: box2
        mu: 0.3
        support_area_inset: 0
      -
        first: ee
        second: box3
        mu: 0.3
        support_area_inset: 0
      -
        first: ee
        second: box4
        mu: 0.3
        support_area_inset: 0

